### PR TITLE
Underdevelopment sign at the start of covid-19 dungeon

### PR DESCRIPTION
**Pull request by mamamia5x**
Just a simple warning to users to let them know that it isn't finished.

### CODE REVIEW

**Comment by paulsonjpaul** [comment]
Good job @mamamia5x. But, I think this will scare off some users and they might leave the dungeon unplayed (I created the dungeon, so that concerns me 😉).

Anyway, there are some parts to change in your code and I have left comments in them.

> Please leave your feedback @ming-tsai

File: english/covid-19/begin-journey.md [hunk 1]
```diff
@@ -2,4 +2,6 @@ The world 🌎 is staying in their homes 🏠 due to the dangerous Covid-19 pand
 
 You walk 🚶‍♂️ out of your house with no precautionary methods to meet a friend 👨. After walking halfway, three sphere-shaped figures 😈 with thorns sticking out their body comes towards you and you don't remember anything that happened the rest of the day 💫.
 
-[🤠 Begin your journey](0/0.md)
\ No newline at end of file
+[🤠 Begin your journey](0/0.md)
+
+🚧 This dungeon is still underdevelopment. You can help us by [contributing](https://github.com/MakeContributions/markdown-dungeon). 🚧
```
Comment (paulsonjpaul) [inline english/covid-19/begin-journey.md:7]:
```suggestion
🚧 This dungeon is still under development. You can help us by [contributing](https://github.com/MakeContributions/markdown-dungeon#-contribution-guidelines). 🚧
```

File: english/covid-19/begin-journey.md [hunk 2]
```diff
@@ -2,4 +2,6 @@ The world 🌎 is staying in their homes 🏠 due to the dangerous Covid-19 pand
 
 You walk 🚶‍♂️ out of your house with no precautionary methods to meet a friend 👨. After walking halfway, three sphere-shaped figures 😈 with thorns sticking out their body comes towards you and you don't remember anything that happened the rest of the day 💫.
 
-[🤠 Begin your journey](0/0.md)
\ No newline at end of file
+[🤠 Begin your journey](0/0.md)
+
+🚧 This dungeon is still under development. You can help us by [contributing](https://github.com/MakeContributions/markdown-dungeon#-contribution-guidelines). 🚧
```
Comment (paulsonjpaul) [inline english/covid-19/begin-journey.md:7]:
```suggestion
🚧 This dungeon is still under development. You can still play it and have fun. To complete the dungeon, you can [contribute](https://github.com/MakeContributions/markdown-dungeon#-contribution-guidelines). 🚧
```

Comment (paulsonjpaul) [reply english/covid-19/begin-journey.md:7]: This will avoid the dungeon being deserted

**Comment by ming-tsai** [comment]
Please use dynamic url @mamamia5x

**Review by Dexters-Hub — APPROVED** [review]

**Review by ming-tsai — APPROVED** [review]

**Pull request merged**